test/url-params.test.ts

```diff
@@ -262,9 +265,48 @@ test('set default client', () => {
 	urlParams.consumeUrlParameters('optOutAnalytics=false');
 	expect(urlParams.isOptOutRequested()).toBeFalsy();
 
+	const passedDeviceId = urlParams.getPassedDeviceId();
 	const client = clientMock();
-	const initialDeviceId = client.deviceId();
+
+	if (passedDeviceId) {
+		client.setDeviceId(passedDeviceId);
+	} else {
+		client.deviceId();
```

Comment:
I believe this `else` statement is redundant here as well as the following test below.

Reply:
Right, I see now how it was redundant since `setClient` should handle that work now.